Arrowgene.Ddon.GameServer/Tasks/DailyTask.cs

```diff
@@ -0,0 +1,23 @@
+using Arrowgene.Ddon.Shared.Model.Scheduler;
+using System;
+
+namespace Arrowgene.Ddon.GameServer.Tasks
+{
+    public abstract class DailyTask : SchedulerTask
+    {
+        public uint Hour { get; }
+        public uint Minute { get; }
+
+        public DailyTask(TaskType scheduleType, uint hour, uint minute) : base(ScheduleInterval.Daily, scheduleType)
+        {
+            Hour = hour;
+            Minute = minute;
+        }
+
+        public override long NextTimestamp()
+        {
+            var tomorrow = DateTime.Today.AddDays(1);
+            return new DateTimeOffset(new DateTime(tomorrow.Year, tomorrow.Month, tomorrow.Day, (int)Hour, (int)Minute, 0)).ToUnixTimeSeconds();
```

Comment:
What timezone does this use?

Reply:
Timezone of the server. Should we make it take a configurable tz value?

Reply:
Either that or making it more explicit, it's a little bit confusing when the 5AM value is hardcoded for the timer